dnf.spec

```diff
@@ -344,6 +347,14 @@ ln -sr  %{buildroot}%{confdir}/vars %{buildroot}%{_sysconfdir}/yum/vars
     popd
 %endif
 
+%if "%{yum_subpackage_name}" == "yum"
+%pretrans yum
```

Comment:
Shell `%pretrans` scriptlets are forbidden in RH/Fedora because it breaks chroot and initial setup installations.

Reply:
Good point, thanks @Conan-Kudo

Reply:
More accurately, shell `%pretrans` scriptlets are blocked on [RH#1032541](https://bugzilla.redhat.com/show_bug.cgi?id=1032541).